osu.Game.Rulesets.Tau/Scoring/TauScoreProcessor.cs

```diff
@@ -4,6 +4,7 @@ namespace osu.Game.Rulesets.Tau.Scoring
 {
     public class TauScoreProcessor : ScoreProcessor
     {
-        public override HitWindows CreateHitWindows() => new TauHitWindows();
+        // Method now obsoleted as of build 2020.1009.0
```

Comment:
We can remove the `TauScoreProcessor` entirely and simply replace it with `ScoreProcessor`, since there's not much this class is doing anymore.

Reply:
We'll need this if we want to change the scoring ratio from the std one. Combos taking up 70% of score is too much imo.